### PR TITLE
l10n: update Czech Translation

### PR DESCRIPTION
**Pull request by AsciiWolf**
The current Czech translation contains a lot of typos and other errors. This pull request (hopefully) fixes them.

### CODE REVIEW

**Comment by piotrdrag** [comment]
I see that you reverted the use of Unicode typography. These days it's recommended to use localized quotation marks and similar: https://wiki.gnome.org/Design/OS/Typography (of course, these are English-specific, in Polish we use „” instead of “”, and I suppose Czech is similar in that regard.)

**Comment by AsciiWolf** [comment]
@piotrdrag: Thanks for notifying me. I have reverted it.

File: po/cs.po [hunk 1]
```diff
@@ -540,46 +539,44 @@ msgid ""
...
-msgstr "Autentizace je vyžadována pro ukončení „$(unit)”"
+msgstr "Autentizace je vyžadována pro ukončení '$(unit)'."
 
 #: ../src/core/dbus-unit.c:590
 msgid "Authentication is required to reset the \"failed\" state of '$(unit)'."
 msgstr ""
-"Autentizace je vyžadována pro resetování chybného („failed”)  stavu "
-"„$(unit)”."
+"Autentizace je vyžadována pro resetování chybného stavu "
+"'$(unit)'."
```
Comment (keszybz) [inline po/cs.po]: Looks like this should be reverted too.

Comment (AsciiWolf) [reply po/cs.po]: I have already reverted it.

Comment (keszybz) [reply po/cs.po]: Hm, you're right. I thought I was looking at the combined diff, but it seems either me or github got confused.

**Pull request merged**